docs/docsite/rst/playbook_guide/playbooks_handlers.rst

```diff
@@ -79,6 +79,38 @@ Tasks can instruct one or more handlers to execute using the ``notify`` keyword.
 In the above example the handlers are executed on task change in the following order: ``Restart memcached``, ``Restart apache``. Handlers are executed in the order they are defined in the ``handlers`` section, not in the order listed in the ``notify`` statement. Notifying the same handler multiple times will result in executing the handler only once regardless of how many tasks notify it. For example, if multiple tasks update a configuration file and notify a handler to restart Apache, Ansible only bounces Apache once to avoid unnecessary restarts.
 
 
+Notifying and loops
+-------------------
+
+Tasks can use loops to notify handlers. This is particularly useful when combined with variables to trigger multiple dynamic notifications.
+
+.. code-block:: yaml
+
+    tasks:
+    - name: Template services
+      ansible.builtin.template:
+        src: "{{ item }}.j2"
+        dest: /etc/systemd/system/{{ item }}.service
+      notify: Restart {{ item }}
```

Comment:
I would also add a comment inside the example, like this:
```suggestion
      # Note: if *any* loop iteration triggers a change, *all* handlers
      # for this loop are run!
      notify: Restart {{ item }}
```

Reply:
:+1: